src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignedCmsTests.cs

```diff
@@ -1156,7 +1156,9 @@ public static void EnsureDataIsolation_NewDocument(bool detached)
 
             // CheckSignature doesn't read the public mutable data
             contentInfo.Content[0] ^= 0xFF;
+#if !NETCOREAPP
```

Comment:
Not sure if this line has any value at all, but thought it would be useful for tests that execute netfx since we seem to still be building tests in that configuration.

Reply:
It shows that the content-type signed attribute is matched against the document encoding during signature verification, not a (historically) mutable property.

Once the value is immutable it matters less, of course.